src/pages/index.js

```diff
@@ -177,30 +295,45 @@ editModalCloseButton.addEventListener("click", () => {
   closeModal(editModal);
 });
 
-editFormElement.addEventListener("submit", handleEditFormSubmit);
-
 // Add Card Event Button Listener
 addCardButton.addEventListener("click", () => {
   openModal(addCardModal);
 });
 
-closeAddCardModalButton.addEventListener("click", () => {
+addCardModalCloseButton.addEventListener("click", () => {
   closeModal(addCardModal);
 });
 
+// Avatar Edit Modal Button Event Listener
+avatarModalButton.addEventListener("click", () => {
+  openModal(avatarModal);
+})
+
+avatarCloseButton.addEventListener("click", () => {
+  closeModal(avatarModal);
+})
+
+avatarForm.addEventListener("submit", handleAvatarSubmit);
+
 // Add Card Submission Function and Listener
 function handleAddCardSubmit(evt) {
+  const name = addCardNameInput.value;
+  const link = addCardLinkInput.value;
   evt.preventDefault();
-  const inputValues = {
-    Name: addCardNameInput.value,
-    Link: addCardLinkInput.value,
-  };
-  const cardElement = getCardElement(inputValues);
-  cardsList.prepend(cardElement);
-  addCardModalForm.reset();
-  disableButton(addCardModalSubmitButton, settings);
-  closeModal(addCardModal);
-  };
-  addCardModalForm.addEventListener("submit", handleAddCardSubmit);
+  function makeRequest() {
+    return api.addCard({ name, link })
+    .then((data) => {
+      const cardElement = getCardElement(data);
+      cardsList.prepend(cardElement);
+      addCardModalForm.reset();
```

Comment:
* Modals don't close after submission in `.then`

Please, close the modal here in `.then`